src/type/preload.ts

```diff
@@ -280,13 +281,40 @@ export type DefaultStyleId = {
   defaultStyleId: number;
 };
 
-export type MinimumEngineManifest = {
+export type supportedFeaturesItem = {
+  type: string;
+  value: boolean;
+  name: string;
+};
+
+export const supportedFeaturesItem = z.object({
+  type: z.string(),
+  value: z.boolean(),
+  name: z.string(),
+});
```

Comment:
特に決まっているわけではないのですが、zod使ってる系objectは最後に`Schema`をつけてるのが多いので、統一させておくのはどうでしょう 👀 
```suggestion
export const supportedFeaturesItemSchema = z.object({
  type: z.string(),
  value: z.boolean(),
  name: z.string(),
});
```